osgp/platform/osgp-adapter-ws-smartmetering/pom.xml

```
@@ -270,6 +270,18 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+    </dependency>
```

Could this be replaced by a newer version (4.0.0) of [Jakarta XML Binding API](https://mvnrepository.com/artifact/jakarta.xml.bind/jakarta.xml.bind-api)?
```suggestion
<dependency>
    <groupId>jakarta.xml.bind</groupId>
    <artifactId>jakarta.xml.bind-api</artifactId>
</dependency>
```